docs/build/networks.md

```diff
@@ -8,25 +8,55 @@ The Msgport protocol is a set of contracts that defines the [core interfaces](./
 This abstract class can have multiple subclasses, known as messaging protocols, which extend its functionality. Each of these subclasses must adhere to the interfaces defined in the Msgport protocol. To be user-friendly, each messaging protocol has 
 its unique contract address across all the networks it supports. The table below shows the canonical cross-chain contract address for each supported network.
 
-|   Name     |  Canonical Cross-chain Contract Address  |
-|------------|--------------------------------------------|
-|   [ORMP](../learn/messaging-protocols/ormp.md)     | 0x0000000005d961F950adA391C1511c92bbc64D9F |
+|  Contract              |  Canonical Cross-chain Deployment Address  |
+|------------------------|--------------------------------------------|
+| SubAPIMultiSig         | 0x22117Db68370590c1031f52a6D1aDE3DCe0cCf9a |
+| [ORMP](../learn/messaging-protocols/ormp.md) | 0xA72d283015c01807bc0788Bf22C1A774bDbFC8fA |
+| Oracle                 | 0x3f938756ceFa33665719Eb528E581FF3f460b7C6 |
+| Relayer                | 0xaC2b224c2E1eD2E8663097a361A05a72d6671C7D |
+| ORMPUpgradeablePort    | 0x8d22f03a675064BFd7509c87206d33730f33e324 |
 
 ## Supported Networks
 
 The ORMP messaging protocol is available on the following main/test networks, which means you can integrate cross-chain capabilities into your application without needing to deploy the underlying messaging protocol yourself. If your desired network is not listed, please feel free to reach out to our team for support and we will be happy to assist you.
 
 ### For Mainnet
 
-- [Ethereum](https://etherscan.io/address/0x0000000005d961F950adA391C1511c92bbc64D9F) ⇋ [Arbitrum](https://arbiscan.io/address/0x0000000005d961F950adA391C1511c92bbc64D9F)
-- [Ethereum](https://etherscan.io/address/0x0000000005d961F950adA391C1511c92bbc64D9F) ⇋ [Darwinia](https://darwinia.subscan.io/account/0x0000000005d961F950adA391C1511c92bbc64D9F)
-- [Arbitrum](https://arbiscan.io/address/0x0000000005d961F950adA391C1511c92bbc64D9F) ⇋ [Darwinia](https://darwinia.subscan.io/account/0x0000000005d961F950adA391C1511c92bbc64D9F)
-- [Arbitrum](https://arbiscan.io/address/0x0000000005d961F950adA391C1511c92bbc64D9F) ⇋ [Blast](https://blastscan.io/address/0x0000000005d961F950adA391C1511c92bbc64D9F)
-- [Polygon](https://polygonscan.com/address/0x0000000005d961F950adA391C1511c92bbc64D9F) ⇋ [Darwinia](https://darwinia.subscan.io/account/0x0000000005d961F950adA391C1511c92bbc64D9F)
-- [Crab](https://crab.subscan.io/account/0x0000000005d961F950adA391C1511c92bbc64D9F) ⇋ [Darwinia](https://darwinia.subscan.io/account/0x0000000005d961F950adA391C1511c92bbc64D9F)
+- Darwinia <> Crab
```

Comment:
1. Please include the corresponding contract explorer link, allowing users to view the specific contract when selecting chains. 
2. Use ⇋ instead of <>.